content/integrations/sumologic.md

```diff
@@ -13,11 +13,12 @@ Enable the Sumo Logic integration to:
 
 ## Installation
 
-There are two parts to the SumoLogic integration, based on the direction of information between the products. You can send alerts and events from Datadog to Sumologic and you can have SumoLogic send data to Datadog.
+Datadog integration with SumoLogic is two-part: send alerts and events from Datadog to Sumologic, or have SumoLogic send data to Datadog. You can enable one or both.
```

Comment:
*Are* users likely to use both? How do we know?

In any case, I don't think it's up to us to tell users they should pick one or use all of the above, but are you saying 'either' makes it sound like they have to make a choice? How about '*each* service can inform the other', that sounds less mutual-exclusiony.